cdap-integration-test/src/main/java/co/cask/cdap/test/remote/RemoteStreamManager.java

```diff
@@ -122,10 +122,10 @@ public void send(Map<String, String> headers, ByteBuffer buffer) throws IOExcept
   }
 
   @Override
-  public List<StreamEvent> getEvents(long startTime, long endTime, int limit) throws IOException {
+  public <T> List<StreamEvent> getEvents(T startTime, T endTime, int limit) throws IOException {
```

Comment:
this allows to pass in anything? Boolean, Date, Dataset... Why not require a string? Or overload the method for long and string?